### ISSUE TITLE
ClusterLoader - Kubemark 5000 flakes

### ISSUE THREAD

**Issue by krzysied** [kubernetes/perf-tests#462]
Since run 1548 kubemark-5000 become flaky.

Link to testgrid: https://testgrid.k8s.io/sig-scalability-kubemark#kubemark-5000

**Comment by krzysied** [comment]
I looks like there are two issues there:
- The api calls latency increased. It happened between run 1527 and 1528.
- [Run 1567](https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-kubemark-gce-scale/1567) has multiple timed out waits on RC deletion.

**Comment by krzysied** [comment]
The changes between run 1527 and run 1528: https://github.com/kubernetes/kubernetes/compare/9fd7a4c2dcbb12...7514c49ec12e7a

**Comment by krzysied** [comment]
Moving run 1567 bug to separate issue: https://github.com/kubernetes/perf-tests/issues/464

**Comment by wojtek-t** [comment]
This one is exactly the same as this Go regression:
https://github.com/kubernetes/kubernetes/issues/75833

So closing as duplicate.
